spec/integrations/posts_index_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe 'posts index', type: :system do
+  let(:user) { User.create(name: 'Tom', bio: 'Teacher from mexico', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?q=80&w=1374&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D') }
+  let(:post1) { Post.create(user:, title: 'Post 1', text: 'Post content') }
+  let(:comment1) { Comment.create(user:, post: post1, text: 'My_comment') }
+  let(:like1) { Like.create(user:, post: post1) }
+  let(:path) { "/users/#{user.id}/posts" }
+
+  before :each do
+    user.save
+    post1.save
+    comment1.save
+    like1.save
+  end
+
+  it 'can see the profile picture of the user' do
+    visit "/users/#{user.id}"
+    photo = find(:css, '.user-photo')
+    expect(photo).to_not be_nil
+  end
```

Comment:
- [ ] As mentioned above, let's kindly refactor this test to make sure that the user's profile picture is displayed on the post index page.